include/inexor/vulkan-renderer/input/gamepad_data.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "glm/detail/qualifier.hpp"
+#include <GLFW/glfw3.h>
+#include <vector>
+
+#define GLM_PRECISION_HIGHP_DOUBLE
+#define GLM_PRECISION_HIGHP_INT
+#include <glm/glm.hpp>
+
+#include <array>
+#include <shared_mutex>
```

Comment:
```suggestion
#include <glm/detail/qualifier.hpp>
#include <GLFW/glfw3.h>

#define GLM_PRECISION_HIGHP_DOUBLE
#define GLM_PRECISION_HIGHP_INT
#include <glm/glm.hpp>

#include <array>
#include <shared_mutex>
#include <vector>
```

Reply:
Ah yes: We have rules for ordering include directives. See https://inexor-vulkan-renderer.readthedocs.io/en/latest/development/engine-design/main.html#coding-style